openedx/core/lib/xblock_builtin/xblock_discussion/utils.py

```diff
@@ -0,0 +1,17 @@
+"""
+Utils for DiscussionXBlock
```

Comment:
@andy-armstrong I forgot to ask last night: What would be a good place to move this to in your opinion? It seems like it could probably live somewhere in `common`, since it could be useful to both Studio and LMS code. However, none of the top-level directories under `common` seem like a good fit. I thought about moving the file to `common/djangoapps/pipeline_js`; but now that the file also defines a function for obtaining CSS dependencies (via https://github.com/edx/edx-platform/pull/13483/commits/2877d907fb27b77f749d20866a9a4f05c65f6389), I'm not sure if `common/djangoapps/pipeline_js` is the best fit.